src/ResourceManager/Dns/Commands.Dns/SetEnv.ps1

```diff
@@ -0,0 +1,21 @@
+﻿Param(
```

Comment:
What is this and where is it used?

Reply:
It's a helper file that we use to record the tests. Without it, it's very cumbersome to set it up.

Reply:
This setup method already exists and is described here: https://github.com/Azure/azure-powershell/blob/preview/documentation/testing-docs/using-azure-test-framework.md.  Please remove this ps1 file.